### DISCUSSION TITLE
"optimization.splitChunks.chunks" can't get some dynamic import modules

### DISCUSSION THREAD

**Discussion by March-Wind** [webpack/webpack#14382] [Q&A]
# Bug report

**What is the current behavior?**
```chunk.name``` can log "main" and "home", but not have "room"

**If the current behavior is a bug, please provide the steps to reproduce.**
repository:
https://github.com/March-Wind/web-template
run it:
1. npm i 
2. npm run build:spa

"optimization.splitChunks.chunks" can't get some dynamic import modules:
```routes.tsx```:

```
const Home = lazy(() => {
    return import(/* webpackChunkName: "home" */ './pages/home/index')
});
const Room = lazy(() => {
    return import(/* webpackChunkName: "room" */  './pages/room/index');
```

webpack.config:(webpack/config/build.spa.ts)

```
optimization:{
             splitChunks:{
                chunks: (chunk) => {
                       console.log(chunk.name);
                       return true
                 },
             }
}
```

**What is the expected behavior?**
```chunk.name``` can log "main" and "home", but not have "room"

**Other relevant information:**
webpack version:  5
Node.js version: 14
Operating System: 
Additional tools:

**Comment by alexander-akait** [comment]
> https://github.com/March-Wind/web-template

404, looks like it is private repo, can you fix it and I will help
